src/resources/AmalgamationShort/BC.ts

```diff
@@ -0,0 +1,42 @@
+import { AmalgamationResourceIF } from '@/interfaces'
+import { FilingCodes, RuleIds } from '@/enums'
+import { AmalgamationShortSteps } from './steps'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const AmalgamationShortResourceBc: AmalgamationResourceIF = {
+  entityType: CorpTypeCd.BC_COMPANY,
+  displayName: GetCorpFullDescription(CorpTypeCd.BC_COMPANY),
+  steps: AmalgamationShortSteps,
+  filingData: [
+    // order matters - see getFilingData()
+    {
+      entityType: CorpTypeCd.BC_COMPANY,
+      filingTypeCode: FilingCodes.AMALGAMATION_HORIZONTAL
+    },
+    {
+      entityType: CorpTypeCd.BC_COMPANY,
+      filingTypeCode: FilingCodes.AMALGAMATION_VERTICAL
+    }
+  ],
```

Comment:
Note the 2 filing data objects here (instead of the usual one). `getFilingData` returns the correct one.